src/components/App/index.tsx

```diff
@@ -87,14 +94,52 @@ export default class App extends Component<Props, State> {
     this.setState({ file: undefined });
   }
 
+  private exposeAPI() {
```

Comment:
I was thinking of putting this into a lazy-loaded component, but it also needs access to the internal state of the `Compress` instance. My current idea would be to get a ref and pass it to the API component. Not pretty, but it would work 🤷‍♂️